.github/workflows/test-snpeff.yml

```diff
@@ -0,0 +1,23 @@
+# This caller workflow builds an image to the "test" stage.
+# Instructions: replace all the <placeholder> stubs in this template with values for your image.
+# Some explanations come from: https://github.com/actions/starter-workflows/blob/main/automation/manual.yml
+
+name: Test snpeff image
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI or when you submit your pull request
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "<snpeff/5.1/Dockerfile"  # Dockerfile path, e.g. 'htslib/1.14/Dockerfile' so that only your image is tested
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+
+  # This job calls a workflow to build the image to the 'test' stage
+  build-to-test:
+    uses: StaPH-B/docker-builds/.github/workflows/build-to-test.yml@master
```

Comment:
If you change this line to: `./.github/workflows/build-to-test.yml` then you'll be able to run the testing workflow on your forked repo.

The way it is currently set is restricted to the StaPH-B code repository